lib/jekyll-mentions.rb

```diff
@@ -2,28 +2,36 @@
 require 'html/pipeline'
 
 module Jekyll
-  class Mentions < Jekyll::Generator
+  class Mentions < Jekyll::Converter
+    MENTIONABLE_EXTS = %w{.html .htm .md .markdown .mkdn .mkd .textile}
```

Comment:
Sort of. At GitHub, at least, html-pipeline relies on [Linguist](https://github.com/github/linguist) to identify whether a file is convertible or not.